rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/BaseController.java

```diff
@@ -87,8 +87,8 @@
         return metaMap;
     }
 
-    protected String getBaseEnvLinkURL() {
```

Comment:
I would like to leave them as protected if someday we move the split the controller in deferent packages